### PR TITLE
PLT-3336 Create authorize view for OAuth Apps (App Center)

### PR DESCRIPTION
**Pull request by enahum**

### CODE REVIEW

File: api/context.go [hunk 1]
```diff
@@ -102,7 +106,9 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	c.RequestId = model.NewId()
 	c.IpAddress = GetIpAddress(r)
 	c.TeamId = mux.Vars(r)["team_id"]
-	h.isApi = IsApiCall(r)
+
+	// this is being set by the handler functions, no need to re-assign here
+	//h.isApi = IsApiCall(r)
```
Comment (coreyhulen) [inline api/context.go]: this makes me nervous?  are we sure about that.

**Comment by coreyhulen** [comment]
minor worry, comment otherwise +1

File: api/context.go (hunk 1 above)
Comment (enahum) [inline api/context.go]: As far as i can tell every route as one of the following:

* ApiAppHandler
* AppHandler
* AppHandlerIndependent
* ApiUserRequired
* ApiUserRequiredActivity
* UserRequired
* ApiAdminSystemRequired
* ApiAppHandlerTrustRequester
* ApiUserRequiredTrustRequester
* ApiAppHandlerTrustRequesterIndependent

and each one of them assign the isApi attribute for the handler

**Comment by jwilander** [comment]
+1

**Pull request merged**